mavros_local_position_pub: '/mavros/setpoint_position/local'
mavros_velocity_pub : '/mavros/setpoint_velocity/cmd_vel'
mavros_local_atual :  '/mavros/local_position/pose'
mavros_state_sub :    '/mavros/state'
mavros_arm :          '/mavros/cmd/arming'
mavros_set_mode :     '/mavros/set_mode'
mavros_battery_sub :  '/mavros/battery'
extended_state_sub:   '/mavros/extended_state'
mavros_global_position_sub: '/mavros/global_position/global'
mavros_pose_target_sub:  '/mavros/setpoint_raw/local'
mavros_set_global_pub: '/mavros/setpoint_position/global'